app/src/main/java/com/example/music_app/ui/helpers/RetrofitAuthHelper.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.music_app.ui.helpers
+
+import com.example.music_app.constants.BASE_URL
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+object RetrofitAuthHelper {
+    private const val baseUrl = BASE_URL
+
+    fun getInstance(): Retrofit {
```

Comment:
Also, why does this method return type is `Retrofit` instead of `RetrofitAuthService` ?